examples/lvgldemo/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ if(CONFIG_EXAMPLES_LVGLDEMO)
     ${CONFIG_EXAMPLES_LVGLDEMO_STACKSIZE}
     MODULE
     ${CONFIG_EXAMPLES_LVGLDEMO}
+    DEPENDS
+    lvgl_nuttx
```

Comment:
let's add cmake support when upgrading lvgl to v9?

Reply:
The idea is to make lvgl v8 working firslty and migrate to v9.0.1 since v9.0.0 is buggy. It should be easier than v8 because lvgl has builtin nuttx support.

Reply:
Ok.